### DISCUSSION TITLE
Erreur 404 lors de la connexion

### DISCUSSION THREAD

**Discussion by supersarma** [FreshRSS/FreshRSS#4677] [Q&A] [answered]
Bonjour,

J'ai installé Fresh sur mon RPI4 et tout allait bien cependant, j'ai des erreurs 404 désormais.
Voici les étapes que je fait :

1. ouvrir ma page domain : redzic.ch
2. Il pointe automatiquement vers redzic.ch/i 
3. Il me demande de me connecter
4. lorsque je clique sur connecter, j'ai une erreur:
- https://redzic.ch/FreshRSS/p/i/?rid=6339d2edec4a0 (le lien affiché après avoir cliqué sur Connexion)
- The requested URL was not found on this server.
5. Si je m'ouvre ensuite simplement la page https://redzic.ch, Fresh s'ouvre en pointant vers https://redzic.ch/i

Petit problème encore.
Lorsque j'ouvre n'importe quoi dans les paramètres de l'application et que je clique simplement sur valider, il me dit qu'il ne trouve pas la page. 

Avez-vous une idée s'il vous plait ?

Merci d'avance pour votre aide.

Voici mon fichier de configuration pour ce host : 

```
 <VirtualHost *:80>
	ServerName www.redzic.ch
	ServerAlias redzic.ch 

	DocumentRoot /var/www/redzic.ch/FreshRSS/p/

	<Directory /var/www/redzic.ch/FreshRSS/p>
		AllowOverride AuthConfig FileInfo Indexes Limit
		Require all granted
	</Directory>

	ErrorLog ${APACHE_LOG_DIR}/redzic_error.log
	CustomLog ${APACHE_LOG_DIR}/redzic_access.log combined
	
	AllowEncodedSlashes On	

</VirtualHost>

<IfModule mod_ssl.c>
<VirtualHost *:443>

        ServerName www.redzic.ch
        ServerAlias redzic.ch
        DocumentRoot /var/www/redzic.ch/FreshRSS/p/

        <Directory /var/www/redzic.ch/FreshRSS/p>
                AllowOverride AuthConfig FileInfo Indexes Limit
                Require all granted
        </Directory>

        ErrorLog ${APACHE_LOG_DIR}/redzic_error.log
        CustomLog ${APACHE_LOG_DIR}/redzic_access.log combined

	AllowEncodedSlashes On

	SSLEngine on
	SSLCompression off	
	SSLCertificateFile /etc/letsencrypt/live/redzic.ch-0001/fullchain.pem
	SSLCertificateKeyFile /etc/letsencrypt/live/redzic.ch-0001/privkey.pem
	Include /etc/letsencrypt/options-ssl-apache.conf

</VirtualHost>

</IfModule>

```

**Comment by Alkarex** [comment, marked as answer]
Bonjour,
Avez-vous peut-être modifié la configuration Apache depuis l'installation de FreshRSS ?
Est-ce une mise à jour depuis une version précédente de FreshRSS ?
Dans tous les cas, regardez dans votre fichier `./data/config.php` la ligne `base_url` qui devrait contenir quelques chose comme `'https://www.redzic.ch'`

**Reply by supersarma** [reply]
Bonjour,

Immense merci pour votre aide. 
Effectivement, l'url de base_url n'était pas bonne et ce pour une raison qui m'échappe car je n'ai pas le souvenir d'avoir touché ce fichier.

En tout cas, merci encore pour l'aide et tout le travail que vous faites.
